app/services/bulk_upload/lettings/year2022/csv_parser.rb

```diff
@@ -1,8 +1,8 @@
 require "csv"
 
 class BulkUpload::Lettings::Year2022::CsvParser
-  MIN_COLUMNS = 134
-  MAX_COLUMNS = 136
+  FIELDS = 134
+  MAX_COLUMNS = 135
```

Comment:
Why is MAX_COLUMNS and not 134? I assume this is max without counting the offset columns?

Reply:
So I originally put 134 but after discussion with @asmega we decided to go for 135 to give a bit of leniency to the user. The thinking is that they might end up with an additional blank column at the end of their file depending on the software they use to prep the csv, and depending on the technical ability of the user this might not be obvious to them.

Reply:
Ah calm calm. Nice!